deltachat-ios/Controller/ChatListController.swift

```diff
@@ -522,6 +534,75 @@ class ChatListController: UITableViewController {
             view.isHidden = false
         }
     }
+    
+    lazy var accountButtonAvatar: InitialsBadge = {
+        let badge = InitialsBadge(size: 40)
+        badge.setColor(UIColor.lightGray)
+        badge.isAccessibilityElement = false
+        return badge
+    }()
+    
+    private let accountButtonUnreadMessageCounter: MessageCounter = {
+        let view = MessageCounter(count: 0, size: 20)
+        view.backgroundColor = DcColors.unreadBadge
+        view.isHidden = true
+        return view
+    }()
+
+
+    private lazy var accountButton: UIBarButtonItem = {
+        // todo make pretty
+        
+        let rect = CGRect(x: 0, y: 0, width: 70, height: 50)
```

Comment:
I would prefer if we set manually constraints when composing this view, just as we do elsewhere. This line would become obsolete in that case.

Reply:
i tweaked size in the latest commit, but it is probably still a bit handwavy @cyBerta feel free to add a commit to change to "manually constraints"